include/SFML/Window/Event.hpp

```diff
@@ -389,6 +389,19 @@ class Event
 
     template <typename T>
     static constexpr bool isEventSubtype = isInParameterPack<T>(decltype (&m_data)(nullptr));
+
+    template <typename Callable, typename... Ts>
+    static constexpr bool isInvokableWithAnyOf(std::variant<Ts...>*)
+    {
+        return (std::is_invocable_v<Callable, const Ts&> || ...);
+    }
+
+public:
+    template <typename Callable>
```

Comment:
```suggestion
    template <typename Handler>
```
Sorry for the churn but I just realized this name makes more sense given the function's name. I'd also just fix this myself to save you the effort but again, I can't edit PR's submitted from someone's `master` branch :(

Reply:
This way I finally set up "trim trailing whitespace" in CLion, so the hustle gave me something good too :D